cves/CVE-2012-0858.yml

```diff
@@ -313,8 +310,8 @@ lessons:
     applies:
     note:
   distrust_input:
-    applies:
-    note:
+    applies: true
+    note: The fix for this is seperating the pointers for memory so if any of the buffer are invalid or their is an error the free still works
```

Comment:
Grammar and Spelling: Fixed below
The fix for this is separating the pointers for memory, so if any of the buffers are invalid or there is an error the free still works.